src/deb/control/templates

```diff
@@ -0,0 +1,29 @@
+Template: [[artifactId]]/contentrepo_db_host
+Type: string
+Default: localhost
+Description: The host that provides the Ambra database
+
+Template: [[artifactId]]/contentrepo_db_password
+Type: password
+Default:
+Description: The password for the user that will access the Ambra database
+
+Template: [[artifactId]]/contentrepo_port
+Type: string
+Default: 8004
+Description: The HTTP port you would like content repo to be accessible from
+
+Template: [[artifactId]]/control_port
+Type: string
+Default: 8003
+Description: The TCP port to use for Tomcat shutdown control port
+
+Template: [[artifactId]]/crepo_storage_location
+Type: string
+Default: mogile
```

Comment:
This is a bit late in the coming, but if you replace this "mogile" string constant with [[crepo.storage.location]] then at least the default run property will match what was used as the build property.